android/src/main/java/com/adyenreactnativesdk/component/base/BaseModule.kt

```diff
@@ -153,7 +154,10 @@ abstract class BaseModule(context: ReactApplicationContext?) : ReactContextBaseJ
     }
 
     override fun onException(exception: CheckoutException) {
-        if (exception is CancellationException || exception.message == "Payment canceled.") {
+        if (exception is CancellationException ||
+            exception is Cancelled3DS2Exception ||
+            exception.message == "Payment canceled."
```

Comment:
Is it possible that the string might change at runtime? If so, is the normal exception sufficient?

Reply:
This is how [GooglePayComponent](https://github.com/Adyen/adyen-android/blob/develop/googlepay/src/main/java/com/adyen/checkout/googlepay/internal/ui/DefaultGooglePayDelegate.kt#L156) throws it..
This can actually be `CancellationException`